AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/codes_for_py/gpu.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/Intel_Extension_For_PyTorch_Hello_World.py b/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/Intel_Extension_For_PyTorch_Hello_World.py
```

Comment:
when do we apply this patch?
should we mention how to use this patch in README?

Reply:
Hi, this patch is prepared when customer want to train with the Intel_Extension_For_Pytorch_GettingStarted.py file on XPU, so they can use this patch to quickly change to xpu demo.
Yeah, if we add this patch, maybe we should also add the instruction in README to tell customer what this is for, I will add it, or do you prefer to remove it?